dataflow/src/backlog/mod.rs

```diff
@@ -221,7 +226,12 @@ impl WriteHandle {
     where
         I: IntoIterator<Item = Record>,
     {
-        self.handle.add(&self.key[..], self.cols, rs)
+        let mem_delta = self.handle.add(&self.key[..], self.cols, rs);
+        if mem_delta > 0 {
+            self.mem_size += mem_delta as usize;
+        } else if mem_delta < 0 {
+            self.mem_size = self.mem_size.checked_sub(mem_delta as usize).unwrap();
```

Comment:
This doesn't seem right? `mem_delta as usize` when `mem_delta < 0`?

Reply:
It's a subtraction of the absolute value, so still correct?

Reply:
My question was more what happens if you cast a negative `isize` into ` usize`.